cli/crates/server/src/udf_builder.rs

```diff
@@ -56,7 +56,7 @@ async fn run_command<P: AsRef<Path>>(
 
     if output.status.success() {
         trace!("'{command_string}' succeeded");
-        Ok(())
+        Ok(output.stdout)
```

Comment:
Correct me if I'm wrong, but does this being populated not depend on a call to `.stdout(Stdio::piped())` which only happens if the user has passed in trace on the CLI?

Reply:
Yes, good point, I disabled tracing for this particular invocation then.